readme.mdown

```diff
@@ -0,0 +1,62 @@
+# Scrivito Portal App
+
+[![CMS: Scrivito](https://img.shields.io/badge/CMS-Scrivito-brightgreen.svg)](https://scrivito.com) [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![CircleCI](https://dl.circleci.com/status-badge/img/gh/Scrivito/scrivito-portal-app/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/Scrivito/scrivito-portal-app/tree/main) [![GitHub last commit](https://img.shields.io/github/last-commit/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app) [![GitHub top language](https://img.shields.io/github/languages/top/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app)
```

Comment:
* [ ] FYI there's one badge that shows "HTML 60%". Not very useful.

Reply:
That is the case, because of the `design/` folder. Maybe its time to move this somewhere else. Ideas?

Reply:
If we no longer need the prototypes and design scripts on a daily basis, maybe a `design` branch than can be rebased to `main` as needed will do?

Reply:
Nice! See https://github.com/Scrivito/scrivito-portal-app/pull/585.

Reply:
With #585 being merged now it looks like this:
<img width="857" alt="Screenshot 2024-09-04 at 12 52 20" src="https://github.com/user-attachments/assets/4da9c3b4-9649-4181-ab6c-335db2763b42">

Also the languages look like this:
![Screenshot 2024-09-04 at 12 51 04](https://github.com/user-attachments/assets/75d3261f-03f3-4f4d-90c9-e9dccfeb3617)